services/app-api/serverless.yml

```diff
@@ -26,6 +26,9 @@ custom:
   reviewerEmail: ${ssm:/configuration/${self:custom.stage}/email/cms_spa_form_cms_email, ssm:/configuration/default/email/cms_spa_form_cms_email, file(resources/ssm-params.yml):Resources.CmsSpaFormCmsEmail.Properties.Value}
   reviewerCHIPEmail: ${ssm:/configuration/${self:custom.stage}/email/cms_spa_form_chip_email, ssm:/configuration/default/email/cms_spa_form_chip_email, file(resources/ssm-params.yml):Resources.CmsSpaFormChipEmail.Properties.Value}
   chipCcEmail: ${ssm:/configuration/${self:custom.stage}/email/cms_chip_form_cc_email, ssm:/configuration/default/email/cms_chip_form_cc_email, file(resources/ssm-params.yml):Resources.CmsSpaFormChipCcEmail.Properties.Value}
+  dmcoEmail: ${ssm:/configuration/${self:custom.stage}/email/cms_dmco_email, file(resources/ssm-params.yml):Resources.CmsDmcoEmail.Properties.Value}
+  dhcbsooEmail: ${ssm:/configuration/${self:custom.stage}/email/cms_dhcbsoo_email, file(resources/ssm-params.yml):Resources.CmsDhcbsooEmail.Properties.Value}
+  dpoEmail: ${ssm:/configuration/${self:custom.stage}/email/cms_dpo_email, file(resources/ssm-params.yml):Resources.CmsDpoEmail.Properties.Value}
```

Comment:
Looks like this default fall-through is how we keep redeployments from overwriting values already in the environment. Nice!